clang/test/SemaHLSL/BuiltIns/GroupMemoryBarrierWithGroupSync-errors.hlsl

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -emit-llvm-only -disable-llvm-passes -verify -verify-ignore-unexpected
```

Comment:
Will forward this comment here about `-verify-ignore-unexpected`: https://github.com/llvm/llvm-project/pull/111209#discussion_r1793906097

Reply:
It's removed now.